src/utils/index.js

```diff
@@ -54,10 +54,14 @@ export function setVersion(version) {
 }
 
 export function getVersion() {
-  if (/version=(\d+\.?\x?)+/.test(location.href)) {
-    return location.href.match(/version=(\d+\.?\x?)+/)[0];
+  try {
+    if (/version=(\d+\.?\x?)+/.test(location.href)) {
+      return location.href.match(/version=((\d+\.?\x?)+)/)[1];
+    }
```

Comment:
Is it possible we find the query part `version=...&apiversion=..` and match the version key-value pair exactly? Instead of using regexp which is brittle.

And, could you explain what is the inputs and outputs of this function?

Reply:
this function is used to get version information from the URL `location.href`, for example: `https://pulsar.apache.org/admin-rest-api/?version=2.11.0&apiversion=v2#operation/getList`

Reply:
> Is it possible we find the query part version=...&apiversion=.. and match the version key-value pair exactly? Instead of using regexp which is brittle.

NO, need to parse the query string yourself

Reply:
OK. I may suggest to use `version=([0-9.x]+)` for comprehension. You can test it locally.

But +1 to merge as is.

Reply:
got it, thanks.

Reply:
https://github.com/apache/pulsar-site/pull/442 @tisonkun PTAL, thanks.